hack/generator/pkg/astmodel/armconversion/convert_from_arm_function_builder.go

```diff
@@ -302,13 +259,20 @@ func (builder *convertFromARMBuilder) convertComplexTypeNameProperty(conversionB
 			destinationType.Name())
 	}
 
+	tok := token.ASSIGN
+	errExists := params.Locals.HasName("err")
+	if !errExists {
```

Comment:
Nice! I think the check is clear enough even without `errExists`.
```suggestion
	if !params.Locals.HasName("err") {
```

Reply:
Fixed!